ast/Trees.h

```diff
@@ -899,6 +899,65 @@ EXPRESSION(Send) {
     // Remove all arguments to the function, including the block argument.
     void clearArgs();
 
+    template <typename PtrT> struct KwPair {
+        absl::Span<PtrT> span;
+
+        PtrT &key() {
+            return span[0];
+        }
+
+        PtrT &value() {
+            return span[1];
+        }
+    };
```

Comment:
We chatted in a DM about making this into `std::pair` instead, or reworking it so that the key and value are references of type `PtrT`, both of which would allow for structured bindings when used in a for-each loop.